documentation/onboard-dpg-in-sdkautomation/README.md

```diff
@@ -0,0 +1,41 @@
+# Service Onboard DPG with Swagger CI Pipeline
+
+Service Onboarding DPG with Swagger CI pipeline can help you get generated SDK and ApiView during the procedure of creating swagger PR.
+
+This Document describes the process of how to onboard DPG with SDK automation pipeline. __This document is targeting for service team.__ 
+
+*If you want to go through details of pipeline implementation, please go to [Integrate DPG and ApiView in SDK Automation Pipeline](Integrate-dpg-and-apiview-in-sdk-automation-pipeline.md).*
+
+## WorkFlow
+
+![workflow-for-service-team](workflow-service-team.png)
+
+__Description:__
+1. Creates a swagger PR.
+2. If the service has onboarded DPG before, please go to step 4. Otherwise, please go to step 3.
+3. Create branch with name `dpg/<prNumber>` in sdk repository, and add `autorest.md` or `README.md`. Then add a comment `/azp run` in swagger PR.
```

Comment:
You may indicate that create the configuration file for SDK generation for each language.

Do you have specific guidance on what folder, what file name and what the file content should be in this step?

Reply:
Yes, there is a guide link in step 3.2, but the guide is not ready.